lib/deck.rb

```diff
@@ -0,0 +1,19 @@
+class Deck
+  attr_reader :cards,
+              :count
+
+  def initialize(cards)
+    @cards = cards
+    @count = cards.count
+  end
+
+  def cards_in_category(topic)
+    array = []
+    @cards.each do |card|
+      if card.category == topic
+        array << card
+      end
+    end
+    return array
+  end
```

Comment:
Nice work here, I'd only suggest naming the variable on line 11 something a bit more descriptive than `array`.